WORKSPACE

```diff
@@ -35,21 +35,25 @@ bind(
 # When updating envoy sha manually please update the sha in istio.deps file also
 #
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
-ENVOY_SHA = "b3be5713f2100ab5c40316e73ce34581245bd26a"
+ENVOY_SHA = "b274f156f066b6221a5319ed1754b7b33b8f3c68"
 
-ENVOY_SHA256 = "79629284ae143d66b873c08883dc6382fac2e8ed45f6f3521f7e7282b6650216"
+ENVOY_SHA256 = "8c47c402fecb397b98bc65c5c340d5c4575f7936f9fa0796a8bf545988dbb5e7"
 
 http_archive(
     name = "envoy",
     sha256 = ENVOY_SHA256,
     strip_prefix = "envoy-" + ENVOY_SHA,
-    url = "https://github.com/envoyproxy/envoy/archive/" + ENVOY_SHA + ".tar.gz",
+    url = "https://github.com/istio/envoy/archive/" + ENVOY_SHA + ".tar.gz",
```

Comment:
Why do we use istio/envoy instead of envoyproxy/envoy? Are we going to actively update istio/envoy daily or weekly?

Reply:
This is temporary, to make sure that we can have this merged in Istio as soon as possible.

The intention is to switch back to upstream once @quanjielin's PR is merged.

Reply:
Now it is merged so just use upstream master?